test/unit/reports/get-status.test.js

```diff
@@ -0,0 +1,26 @@
+const { PAYMENT_ENRICHED_STATUS, PAYMENT_EXTRACTED_STATUS } = require('../../../app/constants/statuses')
+
+const { getStatus } = require('../../../app/reports/mi-report/get-status')
+
+const extractedEvent = require('../../mocks/events/extracted')
+const enrichedEvent = require('../../mocks/events/enriched')
+
+let events
+
+describe('get status', () => {
+  beforeEach(() => {
+    events = []
+  })
+
+  test('should extracted status if extracted last event', () => {
+    events = [extractedEvent]
+    const status = getStatus(events)
+    expect(status).toEqual(PAYMENT_EXTRACTED_STATUS)
+  })
+
+  test('should enriched status if enriched last event', () => {
+    events = [extractedEvent, enrichedEvent]
+    const status = getStatus(events)
+    expect(status).toEqual(PAYMENT_ENRICHED_STATUS)
+  })
```

Comment:
Should these be 'should extract...' and 'should enrich...' respectively? It sounds better to me, but happy to leave as is if you think otherwise

Reply:
Yeah, I think 'should return...' would probably be better

Reply:
Updated